sdk/src/Core/Amazon.Runtime/Credentials/Internal/_bcl45+netstandard/SsoTokenUtils.cs

```diff
@@ -92,6 +92,23 @@ public static string AsJson(this SsoToken token)
             return ToJson(token);
         }
 
+        /// <summary>
+        /// This determines whether the SSO token's registrationExpiresAt field is 5 minutes within expiration
+        /// </summary>
+        /// <param name="token"></param>
+        /// <returns></returns>
+        public static bool RegisteredClientExpired(this SsoToken token)
```

Comment:
nit: It's not _technically_ expired yet if it's within the 5 minute window. Just don't want to mislead somebody attempting to call this in the future if they have a stricter use case. `IsClientRegistrationAboutToExpire` or something like that instead is more accurate.

Reply:
While it's true that it is not technically expired, I think the summary with the intellisense is enough for the user to know it's within 5 minutes.

Reply:
Okay now that you added better XML docs. 👍